src/UI/Implementation/Component/Input/Field/DependantGroupProvider.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace ILIAS\UI\Implementation\Component\Input\Field;
+
+use ILIAS\UI\Component as C;
+
+/**
+ * This is a trait for inputs providing dependant groups, such as checkboxes, e.g.
+ */
+trait DependantGroupProvider {
```

Comment:
Lets name this `DependantGroupHelper`, most traits seemed to be named `Helper` atm.